src/api/app/assets/stylesheets/webui2/main.scss

```diff
@@ -0,0 +1,9 @@
+#personal-navigation ul.nav >li.nav-item > a {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+#personal-navigation ul.nav > li.nav-item:not(:last-child) > a::after {
```

Comment:
and :before? :-)

Reply:
With `::after`, it reproduces what we have in the old interface.